docs/node-operators/how-to/upgrade-network.md

```diff
@@ -168,4 +168,16 @@ In the data node logs, you will see the following message:
 
 ```
 2023-03-02T13:01:04.379+0100	INFO	datanode.service	service/protocol_upgrade.go:36	datanode is ready for protocol upgrade
-```
\ No newline at end of file
+```
+
+### 4. Restarting nodes (for node operators NOT using Visor)
+
+#### 4a. As a node operator if you are running both a Vega node and a data node:
+
+1. Check the latest data node network-history segment block height (`SEGMENT-BLOCK-HEIGHT`) on data node by running: `vega datanode network-history latest-history-segment —home $DATA_NODE_HOME`
+2. Start the data node with the following command: `vega datanode start`
+3. Once the data node has started, proceed to start the Vega core node on the **same block height as data node** with the following command: `vega node —-home $VEGA-NETWORK-HOME --snapshot.load-from-block-height $SEGMENT-BLOCK-HEIGHT`
```

Comment:
Hmmm, I am not sure about those steps if you have a new binary(wanted after protocol upgrade) and you start from an earlier block it will panic, 

With protocol upgrade we should just update binary and restart the network when vega & data node are ready- We know this from logs

Reply:
Yeah actually that is true. Hmm. this was for restart when the network fails. Not a smooth protocol upgrade.

Reply:
I will change it

Reply:
@daniel1302 Actually, I think you need to read the previous steps. there or not part of this PR. We do mentioned that validator should watch for the logs: https://github.com/vegaprotocol/documentation/blob/c09d98c7de7e7dc5fe501a9a2583695e2b892da4/docs/node-operators/how-to/upgrade-network.md#check-if-vega-core-is-ready-for-restart